modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -77,6 +77,31 @@ $ kubectl -n openshift-network-operator logs -l "name=network-operator"
 +
 On high availability clusters with three or more control plane (master) nodes, the Operator will perform leader election and all other Operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
 
+== Addressing the `No disk found with matching rootDeviceHints` error
+
+After the deployment of a cluster you might receive the following error:
+
+----
+`No disk found with matching rootDeviceHints`
+----
+
+To address the "No disk found with matching rootDeviceHints" error, a temporary workaround is to change the `rootDeviceHints` to `minSizeGigabytes: 300`.
+
+After changing the `rootDeviceHints`, boot the CoreOS and then verify the disk information by using the command:
+
+[source]
+----
+$ udevadm info /dev/sda
+----
+
+If you are using DL360 Gen 10 servers, be aware that they have an SD-card slot which might be assigned the `/dev/sda` device name. If no SD card is present in the server, it can cause conflicts. Ensure that the SD card slot is disabled in the server's BIOS settings.
+
+If the `minSizeGigabytes` workaround is not fulfilling the requirements, you might need to revert `rootDeviceHints` back to `/dev/sda`. This change allows ironic images to boot successfully.
+
+An alternative approach to fixing this problem is by using the serial ID of the disk. However, be aware that finding the serial ID can be challenging and might make the configuration file less readable. If you choose this path, ensure that you gather the serial ID correctly using the command `udevadm info /dev/sda` and incorporate it into your configuration.
+
+By following these troubleshooting steps, you can address the issue of "No disk found with matching rootDeviceHints" on your control plane nodes in your {product-title} cluster.
```

Comment:
In the spirit of minimalism, you can consider removing line 103. Its meaning is already implied by the rest of the text in this troubleshooting entry.